Core1Platform/UserInterface/02_Dashboard.md

```diff
@@ -0,0 +1,211 @@
+[!!Manage dashboards](../UsingCore1/05_ManageDashboard.md)
+
+# Dashboards
+
+*Actindo Dashboard*
+
+![Core1 Platform](../../Assets/Screenshots/Core1Platform/Core1.png "[Core1 Platform]")
+
+In the *Actindo Dashboard*, the custom tabs with composable dashlets are displayed. By default no tab is created.
+
+- ![Add](../../Assets/Icons/Plus06.png "[Add]") (Add)   
+    Click this button to add a new tab to display different dashlets. The *Add a new tab* window is displayed.
+
+- ![Delete](../../Assets/Icons/Cross04.png "[Delete]") (Delete)   
+    Click this button to delete the selected tab. The *Confirm tab deletion* window is displayed. This button is displayed to the right of the tab name.
+
+
+
+## Add tab
+
+*Actindo Dashboard > Button Add*
+
+![Create new tab](../../Assets/Screenshots/Core1Platform/UsingCore1/DashboardAddNewTab.png "[Create new tab]")
+
+**Create tab**
+
+- *Tab Name*   
+    Enter a name for the new tab.
+
+- [CANCEL]   
+    Click this button to cancel creating a tab. The *Add a new tab* window is closed.
+
+- [SAVE]   
+    Click this button to save the tab with the entered name. The *Add a new tab* window is closed. The new tab is created.
+
+**Import tab**
+
+![Import tab](../../Assets/Screenshots/Core1Platform/UsingCore1/DashboardImport.png "[Import tab]")
+
+- *Dashboard selection*   
+    Select a module-specific dashboard that you have already created before.   
```

Comment:
Click the drop-down list to select...
already created before -> redundant?

Reply:
done